exonum-java-binding-core/pom.xml

```diff
@@ -22,10 +22,11 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <!-- TODO: stable does not work well until ECR-1839 is resolved. -->
-    <rust.compiler.version>1.26.2</rust.compiler.version>
+    <rust.compiler.version>1.28.0</rust.compiler.version>
```

Comment:
I believe we shall default everywhere to stable, shan't we?

Reply:
:thinking: We've had `stable` written here before.

Reply:
Exactly. And the paths were produced correctly as the command to do that uses rustup/rustc to determine the path.